### ISSUE TITLE
Quick URL/title search needed

### ISSUE THREAD

**Issue by anjackson** [ukwa/shine#63]
The main shine UI is very researcher focussed. We can expect that the first level of search might need to be simpler, in that it focusses on a few main use cases.

* Search for exactly this URL (or very close to it) e.g. for homepage searches.
* Search for this text in the title.
* ...TBA...

I imagine we may be able to achieve this be having a specific search endpoint and Solr config  that uses boosts etc. to raise up 'closer' matches, and to prioritise the 'curated' content over the mass.

We need to work with our user representatives to get examples of the searches terms they would expect to use to get to specific pages or documents, and the kind of search experience they would prefer.

**Comment by peterwebster** [comment]
@anjackson I agree that the current SHINE GUI is very research-oriented, and also that any more "simple" search needs its own GUI presentation - the two won't really mix. Not sure whether I'm going to have any involvement, but some direct user workshopping would be needed, particularly to understand how they would understand/welcome the prioritisation of the curated content.
